app/schemas/schema.js

```diff
@@ -56,9 +58,122 @@ const schema = {
         }
       },
       "type": "object"
+    },
+    "technologies": {
+      "title": "Technologies",
+      "description": "Technologies for which you are intested in submitting applications:",
+      "properties": {
+        "transport": {
+          "type": "array",
+          "title": "Transport:",
+          "items": {
+            "type": "string",
+            "enum": [
+              "Fibre transport",
+              "Microwave transport",
+              "Satellite backhaul"
+            ]
+          },
+          "uniqueItems": true
+        },
+        "lastMile": {
+          "type": "array",
+          "title": "Last-Mile Projects:",
+          "items": {
+            "type": "string",
+            "enum": [
+              "Fibre to the home",
+              "Co-axial cable",
+              "Fixed wireless (LTE preferred)",
+              "Satellite to the home"
+            ]
+          },
+          "uniqueItems": true
+        },
+        "highway": {
+          "type": "array",
+          "title": "Highway cellular:",
+          "items": {
+            "type": "string",
+            "enum": [
+              "Cellular coverage (LTE preferred)",
+              "Rest-stop wi-fi",
+              "Highway call boxes"
+            ]
+          },
+          "uniqueItems": true
+        }
+      },
+      "type": "object"
+    },
+    "preScreen": {
+      "title": "Pre-Screen Information",
+      "description": "How you intend to meet the program's eligibility criteria:",
```

Comment:
@BryanGK everything on this screen is duplicated because I forgot the `"type": "object"` again

Reply:
Nice catch! I noticed that when you add the object type that it also removes some of the titles. I think something worth looking into.